GSA/digitalgov.gov
# Update an-introduction-to-the-digital-experience-council.md

## Summary

Adding Abby Bowman to Digital Experience Leadership Council list.

On on [An introduction to the Digital Experience Council](https://digital.gov/resources/an-introduction-to-the-digital-experience-council/), make the following change:

FROM: [Second co-chair to be selected]

TO: Abby Bowman, co-chair, [Office of the Chief Information Officer](https://www.nasa.gov/ocio/), National Aeronautics and Space Administration

### Preview

[Live site before changes (Before)](https://digital.gov/resources/an-introduction-to-the-digital-experience-council/)
[Link to Preview (After)](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/ss-add-abby-bowman/resources/an-introduction-to-the-digital-experience-council/)



### How To Test

1. Navigate to leadership list under "Who makes up the Digital Experience Council?"
2. Make sure that "[Second co-chair to be selected]" has been replaced with "Abby Bowman, co-chair, [Office of the Chief Information Officer](https://www.nasa.gov/ocio/), National Aeronautics and Space Administration"
3. 
---

### Dev Checklist
